tutorials/applications/QAOA and QAA to solve a MIS problem.ipynb

```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>There's a "Math Processing Error" here, I think because there's a label on this equation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>